src/webpack-mutator.js

```diff
@@ -237,6 +237,10 @@ export default class WebpackMutator extends BaseEvents {
         colors: true
       }
     });
+  }
+
+  serve(cb = () => {}, config) {
+    let server = config || this.getServerConfig();
```

Comment:
@marvelsrp  variable name is **server** but method name is getServer**Config**. (and same for second argument)